crates/protect-ffi/src/lib.rs

```diff
@@ -155,12 +214,18 @@ async fn encrypt_inner(
         )
     })?;
 
-    mp_base85_str_from_encrypted(encrypted)
+    let eql_payload = to_eql_encrypted(encrypted, &ident)?;
+
+    eql_encrypted_to_json_string(&eql_payload)
```

Comment:
`encrypt`/`encryptBulk` return JSON-encoded strings. This object isn't too complex, so I'd like to look at returning a proper JS object here in a follow-up.